packages/webapp/pages/onboarding.tsx

```diff
@@ -48,13 +50,20 @@ const versionToTitle: Record<OnboardingFilteringTitle, string> = {
   [OnboardingFilteringTitle.V4]: 'Choose the topics you’re passionate about',
 };
 
-const Title = classed('h2', 'font-bold typo-title2');
+const Title = classed('h2', 'font-bold typo-title1');
```

Comment:
One important nuance I haven't seen here is that it's an experiment so it should be able to serve a "normal" flow and this new and improved one based on a feature flag.

Reply:
I am working on adding it as an experiment

Reply:
@omBratteng I think this still has a impact on the existing v2 right?

Reply:
It is set to `typo-title2` in `getContent` (V2), so it still had the same size as before. But refactored it a bit so V2 doesn't get both classnames and removed duplicate `font-bold` classname.